metal/mglMetalTest.m

```diff
@@ -20,7 +20,7 @@
 
   mglSetParam('verbose',1);
   if runWithXcode
-    disp(sprintf('Startup mglMetal with XCode now...'))
+    fprintf('Startup mglMetal with XCode now...\n')
```

Comment:
I have no real opinion about which of these to use, but I am generally in favor of accepting default style and formatting suggested by the editor, which is what I did here.